web/connect/result/handler.go

```diff
@@ -0,0 +1,60 @@
+package result
+
+import (
+	"html/template"
+	"net/http"
+
+	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+)
+
+type handler struct {
+	connections  sdkservices.Connections
+	integrations sdkservices.Integrations
+}
+
+func NewHandler(s sdkservices.Services) handler {
+	return handler{connections: s.Connections(), integrations: s.Integrations()}
+}
+
+// ServeHTTP is an HTTP handler that serves generic
+// connection success and error webpages for all integrations.
+func (h handler) ServeHTTP(w http.ResponseWriter, r *http.Request, t *template.Template) {
+	// Read the connection ID from the request path.
+	cid, err := sdktypes.StrictParseConnectionID(r.PathValue("id"))
+	if err != nil {
+		http.Error(w, "Bad connection ID", http.StatusBadRequest)
+		return
+	}
+
+	// Fetch the connection details.
+	conn, err := h.connections.Get(r.Context(), cid)
```

Comment:
I think you need to apply sdk ignoreNotFound. 
Get will return notFound and ignoreNotFound will convert error to nil and object to invalid

Reply:
But I don't want to ignore not-found errors, I need to return an error if it doesn't exist. But if I understand you correctly, I can deletes the `conn.IsValid()` check because it will always be valid if `err == nil`?

Reply:
I meant that if you want to return two distinct errors - internal error and connection not found one, then the way to do it is to use sdkerrors.ignoreNotFound. Then notFound error will be converted to nil and obj to invalid and your code will be valid

Reply:
Done